tests/ui/unstable-feature_bound/unstable-feature-cross-crate-exact-symbol.fail.stderr

```diff
@@ -0,0 +1,11 @@
+error[E0284]: type annotations needed: cannot satisfy `unstable feature: `feat_moo``
```

Comment:
please manually emit a better ambiguity fulfillment error for these goals in `TypeErrCtxt::maybe_report_ambiguity`.